docs/network.md

```diff
@@ -0,0 +1,44 @@
+# Network 
+
+## user-mode network (192.168.5.0/24)
+
+By default Lima only enables the user-mode networking aka "slirp".
+
+### Guest IP (192.168.5.15)
+
+The guest IP address is typically set to 192.168.5.15.
+
+This IP address is not accessible from the host by design.
+
+Use `vde_vmnet` to allow accessing the guest IP from the host and other guests.
+
+### Host IP (192.168.5.2)
```

Comment:
Unrelated to this PR, but I wonder if we shouldn't add an entry to `/etc/hosts` to provide a symbolic name for it?

Challenge is to find a good name, as the meaning of "host" is reversed inside the guest. I guess `hostlocalhost` is rather confusing. 😄

Reply:
The hostname should be like `host.lima.internal` to follow the `host.docker.internal` / `host.containers.internal` convention.

Not in this PR, though.